aspnetcore/test/middleware/samples_snapshot/3.x/false-failure-check.cs

```diff
@@ -0,0 +1,23 @@
+[Fact]
+public async Task MiddlewareTest_ReturnsNotFoundForRequest()
+{
+    using var host = await new HostBuilder()
+        .ConfigureWebHost(webBuilder =>
+        {
+            webBuilder
+                .UseTestServer()
+                .ConfigureServices(services =>
+                {
+                    services.AddMyServices();
+                })
+                .Configure(app =>
+                {
+                    app.UseMiddleware<MyMiddleware>();
+                });
+        })
+        .StartAsync();
+
+    var response = await host.GetTestServer().CreateClient().GetAsync("/");
```

Comment:
Separating this out a little so the first line (GetTestServer) can go in the setup example, and then you have the other two lines for HttpClient example.
```suggestion
    var testServer = host.GetTestServer();
    
    var testClient = testServer.CreateClient();
    var response = await testClient.GetAsync("/");
```